opentelemetry-sdk/tests/trace/test_trace.py

```diff
@@ -1477,7 +1497,8 @@ def _test_span_no_limits(self, tracer):
             OTEL_SPAN_ATTRIBUTE_COUNT_LIMIT: "13",
             OTEL_SPAN_EVENT_COUNT_LIMIT: "7",
             OTEL_SPAN_LINK_COUNT_LIMIT: "4",
-            OTEL_SPAN_ATTRIBUTE_VALUE_LENGTH_LIMIT: "11",
+            OTEL_ATTRIBUTE_VALUE_LENGTH_LIMIT: "11",
+            OTEL_SPAN_ATTRIBUTE_VALUE_LENGTH_LIMIT: "15",
```

Comment:
Which one takes the priority when both are set?

Reply:
Model specific i.e OTEL_SPAN_ATTRIBUTE_VALUE_LENGTH_LIMIT

Reply:
Each model uses its model specific value if set, if not then it uses its default and finally the global one.

Reply:
Would you mind updating the doc to reflect this point?